tools/developer/run.sh

```diff
@@ -161,6 +168,10 @@ function build {
     make -j 16
     #make
     sudo make install
+
+    # Use this one only for macOS Homebrew
+    make install
```

Comment:
Thanks!
I think the `make install` may cause a permission error in Linux/Ubuntu environment, so it might be better to comment it out by default.

This is optional, but for the comment on that line, just `For macOS Homebrew` may be enough.